socialhome/streams/app/components/stamped_elements/ProfileStampedElement.vue

```diff
@@ -0,0 +1,155 @@
+                        :aria-label="translations.organizeProfileContent"
+                    >
+                        <i class="fa fa-arrows-v"></i> {{ translations.organizeProfileContent }}
+                    </b-dropdown-item>
+                </b-dropdown>
+            </div>
+            <div class="mt-1">
+                <b-button
+                    variant="secondary"
+                    :href="urls.contactsFollowed"
+                    :title="translations.following"
+                    :aria-label="translations.following"
+                >
+                    <i class="fa fa-user" />
+                    <i class="fa fa-arrow-right" />
+                    <i class="fa fa-users" />
+                    &nbsp;{{ profile.followingCount }}
+                </b-button>
+            </div>
+            <div class="mt-1">
+                <b-button
+                    variant="secondary"
+                    href="#"
+                    :title="translations.followers"
+                    :aria-label="translations.followers"
+                >
+                    <i class="fa fa-users" />
+                    <i class="fa fa-arrow-right" />
+                    <i class="fa fa-user" />
+                    &nbsp;{{ profile.followersCount }}
+                </b-button>
+            </div>
+        </div>
+        <div class="d-inline-block">
+            <img
+                v-if="profile.saferImageUrlLarge"
+                class="profile-stream-stamped-image"
+                :src="profile.saferImageUrlLarge"
+            />
+        </div>
+        <div class="d-inline-block ml-3 align-center stamped-profile-info">
+            <h1>{{ nameOrGuid }}</h1>
+            <h3><cite :title="translations.userHandle">{{ profile.handle }}</cite></h3>
+        </div>
+        <div class="text-center">
+            <b-button v-if="profile.pinnedContentExists" :variant="pinnedContentVariant" :href="urls.pinnedContent">
+                {{ translations.pinnedContent }}
+            </b-button>
+            <b-button :variant="profileContentVariant" :href="urls.profileAllContent">
+                {{ translations.profileAllContent }}
+            </b-button>
+        </div>
+        <div class="clearfix" />
+    </div>
+</template>
+
+<script>
+import Vue from "vue"
+import applicationStore from "streams/app/stores/applicationStore"
+
+
+export default Vue.component("profile-stamped-element", {
+    computed: {
+        nameOrGuid() {
+            return this.profile.name ? this.profile.name : this.profile.guid
+        },
+        pinnedContentVariant() {
+            return this.profile.streamType === "pinned" ? "primary" : "secondary"
+        },
+        profileContentVariant() {
+            return this.profile.streamType === "all_content" ? "primary" : "secondary"
+        },
+        profile() {
+            return this.$store.state.applicationStore.profile
+        },
+        showAdminButtons() {
```

Comment:
I think `admin` is kind of misleading here. We will have admin actions at some point somewhere - but those will be for site admins. This is really just the persons own profile menu. So maybe `showProfileMenu`?